EmerisHQ/demeris
# fix: Retreive chain_name more reliably

## Description
Makes sure VERIFIED_DENOMS are loaded before retrieving chain names, effectively preventing `/chain/undefined` from being called

Fixes: #1327 

## Feature flags

?VUE_APP_FEATURE_STAKING=1

## Testing
1. Go to https://deploy-preview-1345--emeris-app.netlify.app/asset/uatom?VUE_APP_FEATURE_STAKING=1
2. open the dev tools
3. check whether `/chain/undefined` errors are visible

---

## Additional Details (Optional)

For additional details.

## Reviews (Optional)
@clockworkgr, @eitjuh 

## Screenshots (Optional)

Please attach screenshots if required.
